src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +30,18 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
     }
 
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
```

Comment:
Ha jól látom, csak kitörölted ezt a sort, viszont máshol nem definiáltad a `generator` változót, és emiatt elhasal a build is. Add hozzá az osztályhoz tagváltozóként.